.gitignore

```diff
@@ -1,9 +1,17 @@
+.DS_*
 *.log
-.DS_Store
-node_modules
+logs
+**/*.backup.*
+**/*.back.*
 .cache
-.rts2_cache_cjs
-.rts2_cache_esm
-.rts2_cache_umd
-.rts2_cache_system
+
+node_modules
+bower_componets
 dist
+
+*.sublime*
```

Comment:
Delete the sublime, pad, thumb, and sketch lines. They won't be needed and we don't need to cause any confusion.